c/chosen/chosen_rhel_8_4.sh

```diff
@@ -2,8 +2,9 @@
 #
 # Package       : chosen
 # Version       : 1.0.0
-# Source repo   : https://github.com/harvesthq/chosen/archive/1.0.0.zip
+# Source repo   : https://github.com/harvesthq/chosen
 # Tested on     : UBI 8
+# Language      : HTML
```

Comment:
Use Node as Language for NPMs.

Please check the templates at : https://github.com/ppc64le/build-scripts/blob/master/templates/ for more details.

Reply:
Okay